dashboard/test/models/ability_test.rb

```diff
@@ -797,6 +797,37 @@ class AbilityTest < ActiveSupport::TestCase
     refute Ability.new(student_1).can? :view_as_user_for_code_review, @login_required_script_level, student_2
   end
 
+  test 'only the project owner can create a code review on that project' do
+    project_owner = create :student
```

Comment:
Nit: I know we create everywhere in this file, but can all of these just be `build` instead?

Reply:
I think they can, but I don't see `build` used anywhere in this file and don't really want to introduce it without fixing all the other instances.  (TIL: I thought we just use `create` everywhere but I now see there are pockets of code where we use `build`.)

Reply:
I think a lot of the other ones (probably not all) actually require the records to be created in the database (ie, there's some database lookup happening in the cancan logic), whereas your new logic just looks at the provided objects. I try to follow "build if create not needed", but see where you're coming from.